pymc3/stats.py

```diff
@@ -478,7 +481,7 @@ def gradient(w):
                      bounds=bounds,
                      constraints=constraints)
 
-        weights = w_fuller(w['x'])
+        weights = w_fuller(np.round(w['x'], round_to))
```

Comment:
I'm not sure I like the idea of rounding. Wouldn't this easily round eg the error to 0? Users can use `pd.set_option` (see [here](https://pandas.pydata.org/pandas-docs/stable/options.html)). I guess it might be nice to set a different default for the output dataframe, but I don't think that is possible.